### ISSUE TITLE
Support subreddit

### ISSUE THREAD

**Issue by xaur** [xaur/decred-issues#120]
Benefits I see from a dedicated support subreddit:

* unload r/decred from support requests
* give redditors better support

There is a chance better support would happen on a dedicated subreddit because devs are more likely to watch a place where only support requests are allowed, compared to filtering through a general place like r/decred. Multiple times I stumbled upon unanswered support questions in posts and comments and forwarded them to #support chat room.

Testing it is cheap: https://www.reddit.com/r/dcrsupport/

**Comment by Tsunekazu** [comment]
Not needed now as there's isn't enough activity but for the future, the way to redirect people to the new sub would be to automate the bot moderator to automatically hide AND repost support threads to the new sub.

**Comment by Tsunekazu** [comment]
One more thing; the mods answering questions on the support subreddit could perhaps charge for their work. The main subreddit mods should only charge for 'structural work' imo (CSS redesign, automoderator tinkering, etc.) - not for posting news about DCR.

**Comment by xaur** [comment]
Good point, that looks like regular support staff work that is easier to verify (billing-wise) when it all happens in a dedicated place.
